client/test/e2e/debtors/debtorgroups.spec.js

```diff
@@ -11,7 +11,7 @@ var FU = require('../shared/FormUtils');
 describe('Debtor Groups Management', function () {
   'use strict';
 
-  let initialGroups = 3;
+  let initialGroups = 7;
```

Comment:
Are you rebuilding your database each time?  I think you need to... this should always be 3.